src/Test/XliffValidatorTestCase.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CommentBundle\Test;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\Exception\InvalidResourceException;
+use Symfony\Component\Translation\Loader\XliffFileLoader;
+
+abstract class XliffValidatorTestCase extends TestCase
+{
+    /**
+     * @var XliffFileLoader
+     */
+    protected $loader;
+
+    /**
+     * @var string[]
+     */
+    protected $errors = [];
+
+    protected function setUp(): void
+    {
+        $this->loader = new XliffFileLoader();
+    }
+
+    /**
+     * @dataProvider getXliffPaths
+     */
+    public function testXliff($path)
+    {
+        $this->validatePath($path);
+
+        if (\count($this->errors) > 0) {
+            $this->fail(sprintf('Unable to parse xliff files: %s', implode(', ', $this->errors)));
+        }
+
+        $this->assertCount(
+            0,
+            $this->errors,
+            sprintf('Unable to parse xliff files: %s', implode(', ', $this->errors))
+        );
+    }
+
+    /**
+     * @return array List all path to validate xliff
+     */
+    abstract public function getXliffPaths();
```

Comment:
```suggestion
    abstract public function getXliffPaths(): array;
```
Then `XliffTest` should be updated too.